Packs/FeedElasticsearch/Integrations/FeedElasticsearch/FeedElasticsearch.py

```diff
@@ -210,18 +210,27 @@ def get_demisto_indicators(search, tags, tlp_color):
 
 
 def update_last_fetch(client, ioc_lst):
-    last_calculated_time = None
+    demisto.debug(f"ElasticSearchFeed: Length of the indicators to fetch is: {len(ioc_lst)}")
+    last_calculated_timestamp = None
     last_ids = []
     for ioc in reversed(ioc_lst):
         calculate_time = dateparser.parse(ioc.get(client.time_field))
-        if calculate_time and (not last_calculated_time or calculate_time >= last_calculated_time):
-            last_calculated_time = calculate_time
+        if not calculate_time:
+            demisto.info(f"ioc {ioc.get('name')} if missing {client.time_field}")
+            break
+        calculate_timestamp = int(calculate_time.timestamp() * 1000)
+        if not last_calculated_timestamp or calculate_timestamp == last_calculated_timestamp:
```

Comment:
```suggestion
        if not last_calculated_timestamp or calculate_timestamp >= last_calculated_timestamp:
```
I think it sold be >=, not ==
am I missing something? maybe aUT is needed.

Reply:
It is the same behavior
The time is sorted so we should never encounter cases like this.
There is already a unit test where the last fetched ids have the same time